crypto/base64/base64.cc

```
@@ -78,7 +78,7 @@ int EVP_EncodedLength(size_t *out_len, size_t len) {
   return 1;
 }
 
-EVP_ENCODE_CTX *EVP_ENCODE_CTX_new(void) {
+EVP_ENCODE_CTX *EVP_ENCODE_CTX_new() {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)